### PR TITLE
Bump SDK to 1.0.0-alpha-20170113-2

### PR DESCRIPTION
**Pull request by piotrpMSFT**
@srivatsn @MattGertz I want to keep CLI up to date with SDK builds so we don't get caught by surprise at the last minute. 

@srivatsn can you review?
@MattGertz can we preemptively merge once green?

### CODE REVIEW

**Comment by MattGertz** [comment]
@piotrpMSFT I've lost track -- is this for RC3 or RTW?

**Comment by livarcocc** [comment]
@MattGertz This is for RC3

**Comment by MattGertz** [comment]
@piotrpMSFT @livarcocc I'm going to need more context, then.  What precisely fails if we don't do this?  Does this need to go in with all of the other post-6PM payloads tonight?

**Comment by piotrpMSFT** [comment]
Talked with Matt offline. This was called out in today's shiproom and we're going ahead and merging. We will need another SDK once it's available to take in the final changes, but this brings in as many of those changes as available right now.

File: build/Microsoft.DotNet.Cli.DependencyVersions.props [hunk 1]
```diff
@@ -3,7 +3,7 @@
   <PropertyGroup>
     <CLI_MSBuild_Version>15.1.0-preview-000522-02</CLI_MSBuild_Version>
     <CLI_Roslyn_Version>2.0.0-rc3-61212-03</CLI_Roslyn_Version>
-    <CLI_NETSDK_Version>1.0.0-alpha-20170111-1</CLI_NETSDK_Version>
+    <CLI_NETSDK_Version>1.0.0-alpha-20170113-2</CLI_NETSDK_Version>
     <CLI_WEBSDK_Version>1.0.0-alpha-20170106-1-203</CLI_WEBSDK_Version>
```
Comment (mlorbetske) [inline build/Microsoft.DotNet.Cli.DependencyVersions.props]: Please bump the web sdk to version 1.0.0-alpha-20170113-2-221 as well to get tests passing again

**Comment by piotrpMSFT** [comment]
Thanks @mlorbetske. @MattGertz this is a great example of the sort of issue we wanted to tease out with the early PR.

**Comment by MattGertz** [comment]
Groovy.

**Comment by livarcocc** [comment]
@mlorbetske even with the new SDK, this is still failing with the same symptoms for us.

**Comment by piotrpMSFT** [comment]
@dotnet-bot test this please.

File: build/Microsoft.DotNet.Cli.DependencyVersions.props [hunk 2]
```diff
@@ -3,8 +3,8 @@
   <PropertyGroup>
     <CLI_MSBuild_Version>15.1.0-preview-000522-02</CLI_MSBuild_Version>
     <CLI_Roslyn_Version>2.0.0-rc3-61212-03</CLI_Roslyn_Version>
-    <CLI_NETSDK_Version>1.0.0-alpha-20170111-1</CLI_NETSDK_Version>
-    <CLI_WEBSDK_Version>1.0.0-alpha-20170106-1-203</CLI_WEBSDK_Version>
+    <CLI_NETSDK_Version>1.0.0-alpha-20170113-2</CLI_NETSDK_Version>
+    <CLI_WEBSDK_Version>1.0.0-alpha-20170113-2-221</CLI_WEBSDK_Version>
```
Comment (mlorbetske) [inline build/Microsoft.DotNet.Cli.DependencyVersions.props]: Web SDK version `1.0.0-alpha-20170114-1-223` has the fix

**Comment by piotrpMSFT** [comment]
@mattgertz fyi, SDK issues have been resolved and merged. We're on schedule.

**Comment by MattGertz** [comment]
Awesome, thanks for the update.

**Pull request merged**